dbus/dbus.go

```diff
@@ -132,12 +141,17 @@ func newConnection(createBus func() (*dbus.Conn, error)) (*Conn, error) {
 	return c, nil
 }
 
-func dbusConnection(createBus func() (*dbus.Conn, error)) (*dbus.Conn, error) {
+func dbusConnection(skipAuthHello bool, createBus func() (*dbus.Conn, error)) (*dbus.Conn, error) {
```

Comment:
I don't really like boolean args since they aren't really self-documenting. Would it be unreasonable to have:

```go
func dbusConnection(createBus func() (*dbus.Conn, error)) (*dbus.Conn, error) {
    // stuff
}
func dbusAuthedConnection(createBus func() (*dbus.Conn, error)) (*dbus.Conn, error) {
    dbusConnection(createBus)
    // auth
    // hello
}
```